tests/system/knowledge_base/test_knowledge_base.py

```diff
@@ -466,18 +459,6 @@ def test_connect_after_delete(knowledge_base):
     assert "does not exist or was deleted" in str(e.value)
 
 
-def test_create_with_text_in_indexed_field_raise(index_name,
```

Comment:
Please don't delete this test but instead mark it with `@pytest.mark.skip()`.  
We would still need it for non-serverless index testing